### ISSUE TITLE
last07e offers to upgrade Ubuntu every 5 minutes

### ISSUE THREAD

**Issue by EastEriq** [blumzi/LAST_issues#57]
Enervating....
![image](https://github.com/user-attachments/assets/a7b6514d-fdee-487d-95b1-9617a2825f22)

**Comment by EastEriq** [comment]
Maybe https://askubuntu.com/questions/1435766/how-to-make-ubuntu-stop-offering-to-upgrade-from-20-04-5-to-22-04-1. I haven't tried.

**Comment by EastEriq** [comment]
`/etc/apt/apt.conf.d/20auto-upgrades` says
```
APT::Periodic::Update-Package-Lists "0";
APT::Periodic::Download-Upgradeable-Packages "0";
APT::Periodic::AutocleanInterval "0";
APT::Periodic::Unattended-Upgrade "1";
```
and 
![image](https://github.com/user-attachments/assets/3111be09-bba8-4575-841b-5d1f79f5e38e)
so I presume that the check shouldn't be done, but that is not the case. OTOH
```
ocs@last07e:~/matlab/LAST$ sudo systemctl status apt-daily-upgrade.timer
● apt-daily-upgrade.timer - Daily apt upgrade and clean activities
     Loaded: loaded (/lib/systemd/system/apt-daily-upgrade.timer; enabled; vendor preset: enabled)
     Active: inactive (dead)
    Trigger: n/a
   Triggers: ● apt-daily-upgrade.service

Sep 10 09:19:22 last07e systemd[1]: apt-daily-upgrade.timer: Refusing to start, unit apt-daily-upgrade.service to trigger not loaded.
Sep 10 09:19:22 last07e systemd[1]: Failed to start Daily apt upgrade and clean activities.
```
and this may have something to do with it, if `systemctl` attempts to restart the failing service every 5 minutes.

**Comment by EastEriq** [comment]
Btw `~/.config` belonged to `root`, now changed to `ocs`, maybe it was because of that. I got onto it because I was also getting dialogs about the impossibility of creating `.config/nemo` (nemo is Unity's file browser).

I also had  to do `chown -R ocs ~/.mozilla`, otherwise firefox cold not start because it failed using the default profile. I think I reported that for other machines long time ago,but can't find where at the moment; anyway, again a symptom that some base directories were created with wrong ownership.
